packages/EasyCore/src/Bridge/Symfony/Validator/Constraints/DecimalValidator.php

```diff
@@ -28,7 +28,15 @@ public function validate($value, Constraint $constraint): void
 
         $value = (string) $value;
 
-        $pattern = \sprintf('/^\d+(\.\d{%d,%d})?$/', $constraint->minPrecision, $constraint->maxPrecision);
+        if (\str_contains($value, 'E') === true) {
+            $value = \sprintf('%0.10f', $value);
+        }
+
+        if ($value !== '0') {
+            $value = \rtrim(\rtrim($value, '0'), '.');
```

Comment:
What if `abc000` string passed?

Reply:
It's ok.  `abc` will be checked by regexp